src/background/background.js

```diff
@@ -13,46 +13,29 @@ async function welcomeTab() {
         index: 1
     });
  }
- 
- 
+
  async function addMenuItemProcessInbox() {
     // Setup menu button for reprocessing inbox
     const menu_id = await messenger.menus.create({
```

Comment:
the assignment `const menu_id` is not required anymore.  
Same below.

Reply:
done